src/main/java/seedu/address/ui/SupplyOrderCard.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import seedu.address.model.order.CustomerOrder;
+import seedu.address.model.order.Order;
+import seedu.address.model.order.SupplyOrder;
+
+/**
+ * An UI component that displays information of a {@code Person}.
+ */
+public class SupplyOrderCard extends UiPart<Region> {
+
+    private static final String FXML = "SupplyOrderListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
```

Comment:
It's great that the supplier orders are integrated into a separate window panel from customer orders. With these two window panels, bakery owners can view both orders simultaneously.